x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -37,8 +38,12 @@ import {
 } from '../../../../common/analytics';
 import { LoadingPanel } from '../loading_panel';
 import { getColumnData } from './column_data';
+import { useKibanaContext } from '../../../../../contexts/kibana';
+import { newJobCapsService } from '../../../../../services/new_job_capabilities_service';
+import { getIndexPatternIdFromName } from '../../../../../../application/util/index_utils';
```

Comment:
this goes one level too deep. it doesn't need to go outside `application`

Reply:
Thanks! Updated in 0a79e2a39475beff24b51a40e78f6d6d3e806cca